lsp/completion.go

```diff
@@ -18,68 +18,35 @@ var enumTypeCompletionDocs string
 //go:embed markdown/completion/enumValue.md
 var enumValueCompletionDocs string
 
-var (
-	snippetKind      = protocol.CompletionItemKindSnippet
-	insertTextFormat = protocol.InsertTextFormatSnippet
-
-	verbCompletionItem = protocol.CompletionItem{
-		Label:  "ftl:verb",
-		Kind:   &snippetKind,
-		Detail: stringPtr("FTL Verb"),
-		InsertText: stringPtr(`type ${1:Request} struct {}
-type ${2:Response} struct{}
-
-//ftl:verb
-func ${3:Name}(ctx context.Context, req ${1:Request}) (${2:Response}, error) {
-	return ${2:Response}{}, nil
-}`),
-		Documentation: &protocol.MarkupContent{
-			Kind:  protocol.MarkupKindMarkdown,
-			Value: verbCompletionDocs,
-		},
-		InsertTextFormat: &insertTextFormat,
-	}
+var completionItems = []protocol.CompletionItem{
```

Comment:
Nice idea, but there should be a comment somewhere in the code describing the format.

Reply:
https://github.com/TBD54566975/ftl/pull/1776/files#diff-596afc4cb243f626e08e3f9a1e6b673138da74be0747e3c67fa652e4e765d5d8R32 ?

I can write more on it if you want.

Reply:
Yeah I saw that, but it's mid-code and not really obvious to future engineers wanting to add more snippets. Maybe just move that? Drop it in the next PR, no need for a dedicated one.